cl/visualizations/views.py

```diff
@@ -32,10 +32,14 @@ def render_visualization_page(request, pk, embed):
     status = None
     if viz.deleted:
         status = statuses.HTTP_410_GONE
+        title = u"Visualization deleted by owner"
```

Comment:
Owner feels strange for some reason.  Creator?

Reply:
Sure. Done.